runtime/queries/gotmpl/locals.scm

```diff
@@ -0,0 +1,12 @@
+[
+  (if_action)
+  (range_action)
+  (block_action)
+  (with_action)
+  (define_action)
+] @local.scope
+
+(variable_definition
+  variable: (variable) @local.definition.var)
```

Comment:
```suggestion
  variable: (variable) @local.definition.variable)
```

I'm not sure exactly how Neovim locals queries are structured. I think they only care about the prefix `local.definition`? For Helix locals, since tree-house, we need `@local.definition.<highlight-name>`. (I need to document this 😅)